src/components/ProfilePicture.tsx

```diff
@@ -0,0 +1,19 @@
+import styles from '@/styles/profile.module.css';
+import Image from 'next/image';
+interface Props {
+    pfp: string | undefined,
+    width: number,
+    height: number
+}
+
+export default function ProfilePicture( {pfp, width, height} : Props) {
+    return <Image
+        id={styles.profileIcon}
+        className="m-auto"
```

Comment:
I'd remove the m-auto class from the `ProfilePicture.tsx` component and instead add it to the `#profileIconContainer` in `profile.module.css`, as it messes up the mobile navbar and would be more customizable for future usage.
```suggestion
        className=""
```